packages/app/src/cli/services/bundle.ts

```diff
@@ -0,0 +1,18 @@
+// import {AppInterface} from '../models/app/app.js'
+import {AppInterface} from '../models/app/app.js'
+import {joinPath} from '@shopify/cli-kit/node/path'
+import {zip} from '@shopify/cli-kit/node/archiver'
+import {writeFile} from 'fs/promises'
+
+export async function writeManifestToBundle(app: AppInterface, bundlePath: string) {
+  const appManifest = await app.manifest()
+  const manifestPath = joinPath(bundlePath, 'manifest.json')
+  await writeFile(manifestPath, JSON.stringify(appManifest, null, 2))
+}
```

Comment:
Is it worth doing this now? because with incremental-dev we are going to have different ways to build the manifest for dev and deploy, and it won't be included in the zip file during `dev` 😅

Reply:
What if we send the manifest instead of the app? So that we can send different manifests